2020-04-28 02:59:30,435 [http-nio-8080-exec-1] ERROR com.rahpouyan.rahayi.demo.controller.TavalodController - shod !!!
2020-04-28 03:00:41,312 [http-nio-8080-exec-1] ERROR com.rahpouyan.rahayi.demo.controller.TavalodController - shod !!!
2020-04-28 03:01:15,410 [http-nio-8080-exec-2] ERROR com.rahpouyan.rahayi.demo.controller.TavalodController - shod !!!
2020-04-28 03:29:53,729 [http-nio-8080-exec-1] ERROR com.rahpouyan.rahayi.demo.commom.ExceptionWrapper - وارد کردن دیتای تکراری
2020-04-28 03:30:57,115 [http-nio-8080-exec-1] ERROR com.rahpouyan.rahayi.demo.commom.ExceptionWrapper - وارد کردن دیتای تکراری
2020-04-28 03:32:08,274 [http-nio-8080-exec-1] ERROR com.rahpouyan.rahayi.demo.commom.ExceptionWrapper - وارد کردن دیتای تکراری
2020-04-28 03:33:45,102 [http-nio-8080-exec-1] ERROR com.rahpouyan.rahayi.demo.commom.ExceptionWrapper - وارد کردن دیتای تکراری
2020-04-28 03:37:45,542 [http-nio-8080-exec-2] ERROR com.rahpouyan.rahayi.demo.commom.ExceptionWrapper - وارد کردن دیتای تکراری
2020-04-28 03:38:56,326 [http-nio-8080-exec-1] ERROR com.rahpouyan.rahayi.demo.commom.ExceptionWrapper - وارد کردن دیتای تکراری
2020-04-28 03:38:56,327 [http-nio-8080-exec-1] ERROR com.rahpouyan.rahayi.demo.commom.ExceptionWrapper - وارد کردن دیتای تکراری
2020-04-28 03:44:36,228 [http-nio-8080-exec-1] ERROR com.rahpouyan.rahayi.demo.commom.ExceptionWrapper - وارد کردن دیتای تکراری
2020-04-28 03:46:59,227 [http-nio-8080-exec-3] ERROR com.rahpouyan.rahayi.demo.commom.ExceptionWrapper - وارد کردن دیتای تکراری
2020-04-28 03:48:02,190 [http-nio-8080-exec-2] ERROR com.rahpouyan.rahayi.demo.commom.ExceptionWrapper - وارد کردن دیتای تکراری
2020-04-28 03:55:14,473 [http-nio-8080-exec-1] ERROR com.rahpouyan.rahayi.demo.commom.ExceptionWrapper - وارد کردن دیتای تکراری
2020-04-28 03:57:56,266 [http-nio-8080-exec-1] ERROR com.rahpouyan.rahayi.demo.commom.ExceptionWrapper - وارد کردن دیتای تکراری
2020-04-28 03:57:56,267 [http-nio-8080-exec-1] ERROR com.rahpouyan.rahayi.demo.commom.ExceptionWrapper - وارد کردن دیتای تکراری
2020-04-28 03:58:32,220 [http-nio-8080-exec-4] ERROR com.rahpouyan.rahayi.demo.commom.ExceptionWrapper - وارد کردن دیتای تکراری
2020-04-28 03:58:32,221 [http-nio-8080-exec-4] ERROR com.rahpouyan.rahayi.demo.commom.ExceptionWrapper - وارد کردن دیتای تکراری
2020-04-28 04:01:09,285 [http-nio-8080-exec-2] ERROR com.rahpouyan.rahayi.demo.commom.ExceptionWrapper - database error: could not execute statement; SQL [n/a]; constraint [null]; nested exception is org.hibernate.exception.ConstraintViolationException: could not execute statement
2020-04-28 05:41:38,678 [http-nio-8080-exec-1] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - type is empty
2020-04-28 05:46:28,204 [http-nio-8080-exec-2] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - type is empty
2020-04-28 05:48:18,230 [http-nio-8080-exec-2] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - imageFileName is empty
2020-04-28 05:48:26,914 [http-nio-8080-exec-4] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - jsonImage is empty
2020-04-28 06:03:15,394 [http-nio-8080-exec-3] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - jsonImage is null
2020-04-28 06:16:06,213 [http-nio-8080-exec-2] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - type is null
2020-04-28 06:16:10,361 [http-nio-8080-exec-4] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - could not execute statement; SQL [n/a]; constraint [RAHPOUYANERAHAYI.SYS_C0014720]; nested exception is org.hibernate.exception.ConstraintViolationException: could not execute statement
2020-04-28 06:18:46,524 [http-nio-8080-exec-3] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - ids for this class must be manually assigned before calling save(): com.rahpouyan.rahayi.demo.model.entity.Tavalod; nested exception is org.hibernate.id.IdentifierGenerationException: ids for this class must be manually assigned before calling save(): com.rahpouyan.rahayi.demo.model.entity.Tavalod
2020-04-28 06:20:58,092 [http-nio-8080-exec-1] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - ids for this class must be manually assigned before calling save(): com.rahpouyan.rahayi.demo.model.entity.Tavalod; nested exception is org.hibernate.id.IdentifierGenerationException: ids for this class must be manually assigned before calling save(): com.rahpouyan.rahayi.demo.model.entity.Tavalod
2020-04-28 06:23:22,278 [http-nio-8080-exec-1] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - We need Id
2020-04-28 06:24:20,598 [http-nio-8080-exec-3] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - could not execute statement; SQL [n/a]; constraint [null]; nested exception is org.hibernate.exception.ConstraintViolationException: could not execute statement
2020-04-28 06:24:40,278 [http-nio-8080-exec-5] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - could not execute statement; SQL [n/a]; constraint [null]; nested exception is org.hibernate.exception.ConstraintViolationException: could not execute statement
2020-04-28 06:25:21,790 [http-nio-8080-exec-7] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - could not execute statement; SQL [n/a]; constraint [null]; nested exception is org.hibernate.exception.ConstraintViolationException: could not execute statement
2020-04-28 06:28:03,392 [http-nio-8080-exec-5] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - Object of class [com.rahpouyan.rahayi.demo.model.entity.Tavalod] with identifier [608]: optimistic locking failed; nested exception is org.hibernate.StaleObjectStateException: Row was updated or deleted by another transaction (or unsaved-value mapping was incorrect) : [com.rahpouyan.rahayi.demo.model.entity.Tavalod#608]
2020-04-28 06:28:08,100 [http-nio-8080-exec-7] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - Object of class [com.rahpouyan.rahayi.demo.model.entity.Tavalod] with identifier [608]: optimistic locking failed; nested exception is org.hibernate.StaleObjectStateException: Row was updated or deleted by another transaction (or unsaved-value mapping was incorrect) : [com.rahpouyan.rahayi.demo.model.entity.Tavalod#608]
2020-04-28 06:29:21,108 [http-nio-8080-exec-2] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - Object of class [com.rahpouyan.rahayi.demo.model.entity.Tavalod] with identifier [608]: optimistic locking failed; nested exception is org.hibernate.StaleObjectStateException: Row was updated or deleted by another transaction (or unsaved-value mapping was incorrect) : [com.rahpouyan.rahayi.demo.model.entity.Tavalod#608]
2020-04-28 06:32:48,584 [http-nio-8080-exec-1] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - Object of class [com.rahpouyan.rahayi.demo.model.entity.Tavalod] with identifier [608]: optimistic locking failed; nested exception is org.hibernate.StaleObjectStateException: Row was updated or deleted by another transaction (or unsaved-value mapping was incorrect) : [com.rahpouyan.rahayi.demo.model.entity.Tavalod#608]
2020-04-28 06:35:04,093 [http-nio-8080-exec-2] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - Object of class [com.rahpouyan.rahayi.demo.model.entity.Tavalod] with identifier [608]: optimistic locking failed; nested exception is org.hibernate.StaleObjectStateException: Row was updated or deleted by another transaction (or unsaved-value mapping was incorrect) : [com.rahpouyan.rahayi.demo.model.entity.Tavalod#608]
2020-04-28 06:35:13,142 [http-nio-8080-exec-3] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - Object of class [com.rahpouyan.rahayi.demo.model.entity.Tavalod] with identifier [608]: optimistic locking failed; nested exception is org.hibernate.StaleObjectStateException: Row was updated or deleted by another transaction (or unsaved-value mapping was incorrect) : [com.rahpouyan.rahayi.demo.model.entity.Tavalod#608]
2020-04-28 06:35:14,266 [http-nio-8080-exec-5] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - Object of class [com.rahpouyan.rahayi.demo.model.entity.Tavalod] with identifier [608]: optimistic locking failed; nested exception is org.hibernate.StaleObjectStateException: Row was updated or deleted by another transaction (or unsaved-value mapping was incorrect) : [com.rahpouyan.rahayi.demo.model.entity.Tavalod#608]
2020-04-28 06:36:13,393 [http-nio-8080-exec-7] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - Object of class [com.rahpouyan.rahayi.demo.model.entity.Tavalod] with identifier [608]: optimistic locking failed; nested exception is org.hibernate.StaleObjectStateException: Row was updated or deleted by another transaction (or unsaved-value mapping was incorrect) : [com.rahpouyan.rahayi.demo.model.entity.Tavalod#608]
2020-04-28 06:36:34,935 [http-nio-8080-exec-1] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - Object of class [com.rahpouyan.rahayi.demo.model.entity.Tavalod] with identifier [608]: optimistic locking failed; nested exception is org.hibernate.StaleObjectStateException: Row was updated or deleted by another transaction (or unsaved-value mapping was incorrect) : [com.rahpouyan.rahayi.demo.model.entity.Tavalod#608]
2020-04-28 06:40:05,161 [http-nio-8080-exec-2] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - Row was updated or deleted by another transaction
2020-04-28 06:40:44,630 [http-nio-8080-exec-5] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - month is null
2020-04-28 06:40:44,640 [http-nio-8080-exec-5] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - Row was updated or deleted by another transaction
2020-04-28 06:40:57,718 [http-nio-8080-exec-7] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - month is null
2020-04-28 06:43:02,363 [http-nio-8080-exec-1] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - month is null
2020-04-28 06:43:06,736 [http-nio-8080-exec-3] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - Row was updated or deleted by another transaction
2020-04-28 06:43:11,924 [http-nio-8080-exec-5] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - Row was updated or deleted by another transaction
2020-04-28 06:43:19,658 [http-nio-8080-exec-8] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - Row was updated or deleted by another transaction
2020-04-29 03:57:55,697 [http-nio-8080-exec-3] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - ids for this class must be manually assigned before calling save(): com.rahpouyan.rahayi.demo.model.entity.Tavalod; nested exception is org.hibernate.id.IdentifierGenerationException: ids for this class must be manually assigned before calling save(): com.rahpouyan.rahayi.demo.model.entity.Tavalod
2020-04-29 04:00:17,717 [http-nio-8080-exec-1] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - ids for this class must be manually assigned before calling save(): com.rahpouyan.rahayi.demo.model.entity.Tavalod; nested exception is org.hibernate.id.IdentifierGenerationException: ids for this class must be manually assigned before calling save(): com.rahpouyan.rahayi.demo.model.entity.Tavalod
2020-04-29 04:00:20,325 [http-nio-8080-exec-4] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - ids for this class must be manually assigned before calling save(): com.rahpouyan.rahayi.demo.model.entity.Tavalod; nested exception is org.hibernate.id.IdentifierGenerationException: ids for this class must be manually assigned before calling save(): com.rahpouyan.rahayi.demo.model.entity.Tavalod
2020-04-29 04:02:11,145 [http-nio-8080-exec-2] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - ids for this class must be manually assigned before calling save(): com.rahpouyan.rahayi.demo.model.entity.Tavalod; nested exception is org.hibernate.id.IdentifierGenerationException: ids for this class must be manually assigned before calling save(): com.rahpouyan.rahayi.demo.model.entity.Tavalod
2020-04-29 04:03:43,477 [http-nio-8080-exec-1] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - ids for this class must be manually assigned before calling save(): com.rahpouyan.rahayi.demo.model.entity.Tavalod; nested exception is org.hibernate.id.IdentifierGenerationException: ids for this class must be manually assigned before calling save(): com.rahpouyan.rahayi.demo.model.entity.Tavalod
2020-04-29 04:04:14,434 [http-nio-8080-exec-4] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - type is null
2020-04-29 05:24:43,955 [http-nio-8080-exec-1] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - ids for this class must be manually assigned before calling save(): com.rahpouyan.rahayi.demo.model.entity.Image; nested exception is org.hibernate.id.IdentifierGenerationException: ids for this class must be manually assigned before calling save(): com.rahpouyan.rahayi.demo.model.entity.Image
2020-04-29 05:31:02,958 [http-nio-8080-exec-2] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - ids for this class must be manually assigned before calling save(): com.rahpouyan.rahayi.demo.model.entity.Image; nested exception is org.hibernate.id.IdentifierGenerationException: ids for this class must be manually assigned before calling save(): com.rahpouyan.rahayi.demo.model.entity.Image
2020-04-29 05:32:13,830 [http-nio-8080-exec-7] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - month is null
2020-04-29 05:32:18,622 [http-nio-8080-exec-9] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - day is null
2020-04-29 05:32:21,474 [http-nio-8080-exec-1] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - name is null
2020-04-29 05:32:23,883 [http-nio-8080-exec-3] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - day is null
2020-04-29 05:32:26,656 [http-nio-8080-exec-5] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - month is null
2020-04-29 05:32:29,482 [http-nio-8080-exec-7] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - could not execute statement; SQL [n/a]; constraint [RAHPOUYANERAHAYI.SYS_C0014720]; nested exception is org.hibernate.exception.ConstraintViolationException: could not execute statement
2020-04-29 05:33:30,201 [http-nio-8080-exec-2] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - could not execute statement; SQL [n/a]; constraint [RAHPOUYANERAHAYI.SYS_C0014720]; nested exception is org.hibernate.exception.ConstraintViolationException: could not execute statement
2020-04-29 05:44:40,040 [http-nio-8080-exec-3] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - could not execute statement; SQL [n/a]; constraint [null]; nested exception is org.hibernate.exception.ConstraintViolationException: could not execute statement
2020-04-29 05:45:50,536 [http-nio-8080-exec-2] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - type is null
2020-04-29 05:46:44,980 [http-nio-8080-exec-5] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - imageFileName is null
2020-04-29 05:47:07,217 [http-nio-8080-exec-7] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - could not execute statement; SQL [n/a]; constraint [RAHPOUYANERAHAYI.SYS_C0014720]; nested exception is org.hibernate.exception.ConstraintViolationException: could not execute statement
2020-04-29 05:50:03,091 [http-nio-8080-exec-1] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - images repeted
2020-04-29 06:00:06,388 [http-nio-8080-exec-2] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - Row was updated or deleted by another transaction
2020-04-29 06:01:35,151 [http-nio-8080-exec-7] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - imageFileName is null
2020-04-29 06:01:40,448 [http-nio-8080-exec-8] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - could not execute statement; SQL [n/a]; constraint [RAHPOUYANERAHAYI.SYS_C0014742]; nested exception is org.hibernate.exception.ConstraintViolationException: could not execute statement
2020-04-29 06:03:29,388 [http-nio-8080-exec-1] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - recordVersion is null
2020-04-29 06:03:33,109 [http-nio-8080-exec-3] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - Row was updated or deleted by another transaction
2020-04-29 06:20:29,444 [http-nio-8080-exec-1] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - Row was updated or deleted by another transaction
2020-04-29 06:21:34,716 [http-nio-8080-exec-5] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - recordVersion is null
2020-04-29 06:22:33,602 [http-nio-8080-exec-9] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - type is null
2020-04-29 06:25:18,311 [http-nio-8080-exec-6] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - name is null
2020-04-29 06:25:22,420 [http-nio-8080-exec-8] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - type is null
2020-04-29 06:25:29,510 [http-nio-8080-exec-10] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - imageFileName is null
2020-04-29 06:25:33,704 [http-nio-8080-exec-1] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - day is null
2020-04-29 06:25:38,163 [http-nio-8080-exec-4] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - month is null
2020-04-29 06:25:41,037 [http-nio-8080-exec-6] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - year is null
2020-04-29 06:25:43,799 [http-nio-8080-exec-8] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - jsonImage is null
2020-04-29 06:26:23,087 [http-nio-8080-exec-10] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - this api need Id
2020-04-29 06:26:28,204 [http-nio-8080-exec-1] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - releaseDate is null
2020-04-29 06:26:31,817 [http-nio-8080-exec-4] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - releaseTime is null
2020-04-29 06:26:37,085 [http-nio-8080-exec-5] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - recordVersion is null
2020-04-29 06:26:41,448 [http-nio-8080-exec-7] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - Row was updated or deleted by another transaction
2020-04-29 06:27:23,162 [http-nio-8080-exec-2] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - Row was updated or deleted by another transaction
2020-04-29 06:27:46,243 [http-nio-8080-exec-5] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - this api need Id
2020-04-30 03:36:41,884 [http-nio-8080-exec-1] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - could not execute statement; SQL [n/a]; constraint [RAHPOUYANERAHAYI.SYS_C0014759]; nested exception is org.hibernate.exception.ConstraintViolationException: could not execute statement
2020-04-30 03:36:55,261 [http-nio-8080-exec-4] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - name is null
2020-04-30 03:36:58,680 [http-nio-8080-exec-5] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - name is null
2020-04-30 03:37:01,462 [http-nio-8080-exec-9] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - day is null
2020-04-30 03:37:47,668 [http-nio-8080-exec-1] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - day is null
2020-05-01 01:11:06,081 [http-nio-8080-exec-7] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - could not execute statement; SQL [n/a]; constraint [null]; nested exception is org.hibernate.exception.ConstraintViolationException: could not execute statement
2020-05-01 01:13:44,158 [http-nio-8080-exec-4] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - could not execute statement; nested exception is org.hibernate.exception.GenericJDBCException: could not execute statement
2020-05-01 01:15:42,414 [http-nio-8080-exec-6] ERROR com.rahpouyan.rahayi.demo.commom.CustomException - could not execute statement; nested exception is org.hibernate.exception.GenericJDBCException: could not execute statement
